src/components/UserInfo/UserInfo.jsx

```diff
@@ -1,3 +1,9 @@
-export const UserInfo = () => (
-  <>Put the user here</>
+import './UserInfo.scss';
+
+export const UserInfo = ({ user }) => (
+  <>
+    <a className="UserInfo" href={`mailto:${user.email}`}>
+      {user.name}
+    </a>
+  </>
```

Comment:
```suggestion

    <a className="UserInfo" href={`mailto:${email}`}>
      {name}
    </a>

```
No need to use fragments here since you have a common parent element. Also, consider restructuring all needed properties